README.md

```diff
@@ -33,6 +33,7 @@ _**Note:** The behavior of actions like this one is currently limited in the con
   - [Black](https://black.readthedocs.io)
   - [Flake8](http://flake8.pycqa.org)
   - [Mypy](https://mypy.readthedocs.io/)
+  - [OITNB](https://pypi.org/project/oitnb/)
```

Comment:
On the project page I only see it written as `oitnb` in all lowercase. What do you think about using the same style here too?

Only for the linter class it would be `Oitnb`.

Reply:
yeah, I'm not sure what the proper capitalization should be as it's an acronym. I will change them around to be consistent with the other class names and the project page

Reply:
done in 626a4f2